src/sentry/sentry_apps/api/bases/sentryapps.py

```diff
@@ -49,6 +49,17 @@ def wrapped(self, *args, **kwargs):
     return wrapped
 
 
+def _handle_sentry_app_exception(exception: Exception):
+    # If the error_type attr exists we know the error is one of SentryAppError or SentryAppIntegratorError
+    if isinstance(exception, SentryAppIntegratorError) or isinstance(exception, SentryAppError):
+        response = Response({"error": str(exception)}, status=exception.status_code)
```

Comment:
i think we usually put the text as detail?

```suggestion
        response = Response({"detail": str(exception)}, status=exception.status_code)
```

should we also add event id? https://github.com/getsentry/sentry/blob/05fa4022efb8719d7527559db993f3777d7a37ea/src/sentry/api/base.py#L333-L335

Reply:
No since these errors are the 'fault' of the integrator/user not us so we don't want to capture them for Integrator & AppError. If it's our fault we capture the event id and send it back which is intended.

We'll have some rollout/testing period later on with the webhook so we can calibrate if the classification is wrong or smthn